src/app/analytics/loyalties/page.tsx

```diff
@@ -1,9 +1,33 @@
-"use client"
+import { api } from "~/trpc/server";
+import ClientPlot from "../revenue/client-plot";
+
+const Page = async () => {
+  const loyalties: number[] = [0, 1, 2, 3];
+  const categoriesByLoyalty =
+    await api.analytics.getLoyaltyRevenueByCategory.query();
+
+  const categoryData = Object.keys(categoriesByLoyalty).map((category) => {
+    return {
+      x: loyalties,
+      y: categoriesByLoyalty[category],
+      name: `${category}`,
```

Comment:
why not just category? i.e.
```suggestion
      name: category,
```